easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.1.0-GCCcore-10.2.0.eb

```diff
@@ -37,11 +37,14 @@ dependencies = [
 ]
 
 configopts = "--enable-ld-version-script "
-configopts += '--disable-webp --disable-libdeflate '
+configopts += "--disable-webp --disable-libdeflate "
 
 sanity_check_paths = {
-    'files': ['bin/tiffinfo'],
+    'files': ['bin/tiffdump', 'bin/tiffinfo', 'include/tiff.h', 'lib/libtiff.a', 'lib/libtiff.%s' % SHLIB_EXT,
+              'lib/libtiffxx.a', 'lib/libtiffxx.%s' % SHLIB_EXT, 'lib/pkgconfig/libtiff-4.pc'],
     'dirs': [],
 }
 
+sanity_check_commands = ["tiffinfo -h"]
```

Comment:
-h flag doesn't exist for 4.1.0 it seems

Reply:
I reverted the sanity check commands for you.

Reply:
Thanks @Micket!